### ISSUE TITLE
Encrypting the Database

### ISSUE THREAD

**Issue by andrewsbryanc** [tbixler1/COSC-481W-Project-Scheduling-#18]
This will be towards the end of the database creation, and will be handy to have if we have extra time towards the end of the semester.  I'm creating the issue now so we have a goal in mind.

**Comment by tgautz** [comment]
Sounds good.  One (quick) idea is to use the ASCII/Unicode values, and just add/subtract to the values to get an encrypted file.  We're not storing credit cards or anything, so I don't think we need a high security encryption.
Also, it would be nice to have a static method for this.  Something like void Encryption.encrypt(complete file path) and void Encryption.encrypt(complete file path).
Just my 2 cents on the matter.

**Comment by andrewsbryanc** [comment]
This is true, I planned on making the encryption super simple, that or use a premade AES package encryption code to make it work.
